src/runtime/config/configuration-qemu-snp.toml.in

```diff
@@ -0,0 +1,665 @@
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# XXX: WARNING: this file is auto-generated.
+# XXX:
+# XXX: Source file: "@CONFIG_QEMU_IN@"
+# XXX: Project:
+# XXX:   Name: @PROJECT_NAME@
+# XXX:   Type: @PROJECT_TYPE@
+
+# Basing SNP updates on this post from Joana
+# https://github.com/kata-containers/kata-containers/blob/main/docs/how-to/how-to-run-kata-containers-with-SNP-VMs.md
+
+[hypervisor.qemu]
+path = "@QEMUSNPPATH@"
+kernel = "@KERNELSEVPATH@"
+#SNP should be able to use SEV kernel and initrd with things being passed via kernel params
+initrd = "@INITRDSEVPATH@"
+machine_type = "@MACHINETYPE@"
+
+# Enable confidential guest support.
+# Toggling that setting may trigger different hardware features, ranging
+# from memory encryption to both memory and CPU-state encryption and integrity.
+# The Kata Containers runtime dynamically detects the available feature set and
+# aims at enabling the largest possible one, returning an error if none is
+# available, or none is supported by the hypervisor.
+#
+# Known limitations:
+# * Does not work by design:
+#   - CPU Hotplug 
+#   - Memory Hotplug
+#   - NVDIMM devices
+#
+# Default false
+confidential_guest = true
+
+# enable SEV SNP VMs
+sev_snp_guest = @DEFSNPGUEST@
+
+# SNP guest policy
+# From Niteesh's https://github.com/kata-containers/kata-containers/pull/5796/files
+# (this will depend on that patch)
```

Comment:
We can remove this line as the [PR5796](https://github.com/kata-containers/kata-containers/pull/5796) is already merged to CCv0.